packages/ai-mcp/tests/client.test.ts

```
@@ -0,0 +1,77 @@
+// packages/ai-mcp/tests/client.test.ts
+import { describe, expect, it } from 'vitest'
+import { makeServerWithWeatherTool } from './helpers/in-memory-server'
```

ESLint

[error] `./helpers/in-memory-server` import should occur after import of `../src/errors`

(import/order)